multimedia/M2VDownsizer/Portfile

```diff
@@ -0,0 +1,78 @@
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                M2VDownsizer
+version             1.1
+revision            0
+
+categories          multimedia
+platforms           darwin
+license             GPL-2
+maintainers         {hotmail.com:amtor @RobK88} \
+                    openmaintainer
+
+description         MPEG-2 video stream recompressesion
+
+long_description    ${name} is a tool that re-compresses MPEG-2 (or m2v) video \
+                    streams.  In other words, ${name} will perform a complete \
+                    decompression and recompression of MPEG-2 video streams for the \
+                    best possible video quality.\
+                    \n \
+                    \n${name}  accepts the raw MPEG-2 (or m2v) video data (not VOB) \
+                    from the standard input and writes the recompressed video frames \
+                    to the standard output.\
+                    \n \
+                    \nBe  forewarned!  Decompression  and  recompression  could take \
+                    hours for large MPEG-2 video streams!  It is recommended that one \
+                    first tries transcoding MPEG-2 streams using M2VRequantiser since \
+                    transcoding is a lot faster and produces excellent results for most \
+                    MPEG-2 video streams. ${name} is only intended to be used for \
+                    large MPEG-2 video streams when transcoding does not produce \
+                    acceptable results.
+
+homepage            https://web.archive.org/web/20120508050929if_/http://www.metakine.com/
+master_sites        ${homepage}/files
+
+depends_lib         port:libmpeg2
+
+checksums           rmd160 d662bfe195b59f5815bfdccaf3f81dae46d4e0d8 \
+                    sha256 316a931810116d70a07cb57a80ae8483fa949bd575beee222a9a8e375c4d8590 \
+                    size 371928
+
+distname            ${name}
+
+dist_subdir         ${name}/${version}
+
+extract.suffix      .tgz
+
+post-extract {
+     copy ${filespath}/Makefile ${worksrcpath}
+     copy ${filespath}/${name}.1 ${worksrcpath}
+     file mkdir ${worksrcpath}/obj/utils
+     file mkdir ${worksrcpath}/obj/altivec
+     file mkdir ${worksrcpath}/obj/mpeg2enc
+     file mkdir ${worksrcpath}/obj/main
+}
+
+patchfiles          1001-Replace-MPProcessors-Patch.diff \
+                    1002-libmpeg2-Linkage-Patch.diff \
+                    1003-Patch-config.h.diff \
+                    1004-Fix-Includes-in-mjpeg_logging.diff
+
+#
+# Turning off parallel builds prevents garbled output from the compiler
+# caused by the following warning messages being generated at
+# the same time when non-ppc builds are being compiled:
+#
+# clang: warning: argument unused during compilation: '-Xarch_ppc -faltivec'
+# clang: warning: argument unused during compilation: '-Xarch_ppc64 -faltivec'
```

Comment:
You could add `-Wno-unused-command-line-argument` to `configure.cflags` to suppress this warning.

Reply:
I will  have to try that first.  I did try `-Wno-unused` but that did not suppress the warnings.  

If `-Wno-unused-command-line-argument` does the trick, it should definitely be added to CFLAGS.  I will try it tomorrow.

Reply:
Thank-you Ryan!  I added `-Wno-unused-command-line-argument` to CFLAGS in the Makefile and it did eliminate all those pesky clang warnings!! 

So I propose that the following be changed in the Makefile:

`CFLAGS += -Wall -g`

be changed to:

`CFLAGS += -Wall -g -Wno-unused-command-line-argument`

And the comment in the Portfile regarding parallel builds should be removed.

Reply:
Please note the flag `"-Wno-unused-command-line-argument"` appears to be only available when using `clang`. This flag does not appear to be available when using `gcc`.

So adding the following to the Portfile is preferable since it may be possible to restrict its use to certain compilers:

`configure.cflags-append      "-Wno-unused-command-line-argument"`

Reply:
This issue has been resolved in my last push.  The flag **-Wno-unused-command-line-argument** has been added to the Makefile.